src/sections/Meshmap/index.js

```diff
@@ -0,0 +1,231 @@
+                Developer
+              </label>
+              <br></br>
+              <label>
+                <Field type="radio" name="role" value="DevOps" />
+                DevOps/SRE/Sysadmin
+              </label>
+              <br></br>
+              <label>
+                <Field type="radio" name="role" value="Executive" />
+                Executive 
+              </label>
+              <br></br>
+              <label>
+                <Field type="radio" name="role" value="IT Operations" />
+                IT Operations
+              </label>
+              <br></br>
+              <label>
+                <Field type="radio" name="role" value="Media" />
+                Media/Analyst
+              </label>
+              <br></br>
+              <label>
+                <Field type="radio" name="role" value="Product Manager" />
+                Product Manager
+              </label>
+              <br></br>
+              <label>
+                <Field type="radio" name="role" value="Professor" />
+                Professor/Academic
+              </label>
+              <br></br>
+              <label>
+                <Field type="radio" name="role" value="Sales Marketing" />
+                Sales Marketing
+              </label>
+              <br></br>
+              <label>
+                <Field type="radio" name="role" value="Student" />
+                Student
+              </label>
+              <br></br>
+              <label>
+                <Field type="radio" name="role" value="Other" />
+                Other
+              </label>
+              <br></br>
+            </div>
+            <br></br>
+            <p className="para"> Choose between Twitter, Google, LinkedIn, and GitHub, provide the username/handle of your user account for your preferred identity provider. Selected beta program participants will receive a free Meshery Cloud account and have full access MeshMap enabled for each of the following user accounts that you provide. Please provide at least one account.</p>
+            <img className="image" src={mesheryCloud} />
+            <Button secondary type="submit" className="btn" title="Submit" /> <br /><br /><br /><br />
+          </Form>
+        </Formik>
+      </Container>
+    );
+  };
+  const ThankYou = () => {
+    return (
+      <Container>
+        <div className="black-box">
+          <h2>Thank you for your interest in MeshMap.</h2>
+          <p>Please patiently await your acceptance into and start of  the beta program. You are now signed up for the Layer5 MeshMap beta program and your position on the waiting list is confirmed. We are working through a growing waitlist. <br />
+If you have any questions in the meantime, please email meshmap@layer5.io.<br />
+Regards,<br />
+            <img src={layer5_img} alt="Layer5" width="85" /> Team
+          </p>
+        </div>
+      </Container>
+    );
+  };
```

Comment:
Since the 2 components `ThankYou` and `MeshmapFormPage` are individual components, define them outside the `MeshMap` component. (In the same file only.)

Reply:
@Nikhil-Ladha actually I've used a hook in the MeshMap component `const [stepNumber, setStepNumber] = useState(0)` which allots each component a step number which I have used to call the components [here](https://github.com/layer5io/layer5/pull/2368/files#diff-48f16f82f3aa3664d25261c20b28eb013aee5be2c6ae0404e225639011076e21R216)

Reply:
That's fine, But, that hook is not dependent on any of the values in the other 2 components. So, they can be defined outside the parent component.

Reply:
Oh no, wait....I think there seems to be some confusion. Ignore this change.